testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AvailableJavaHomes.java

```diff
@@ -184,19 +235,29 @@ public static Jvm getJdk(final JavaVersion version) {
     }
 
     /**
-     * Returns a JDK for each of the given java versions, if available.
+     * Return a list of JDK installations, containing one installation per version
+     * in the specified range, if such a version is available on this machine.
      */
-    public static List<Jvm> getJdks(final String... versions) {
-        List<JavaVersion> javaVersions = Arrays.stream(versions).map(JavaVersion::toVersion).collect(Collectors.toList());
-        return getJdks(Iterables.toArray(javaVersions, JavaVersion.class));
+    public static List<Jvm> getJdksInRange(@Nullable Integer minVersionInclusive, @Nullable Integer maxVersionExclusive) {
```

Comment:
💭 It might be better to use Guava's `Range` here? It seems like it would help with calling it properly.